transformations/insert_abbreviation/test.json

```diff
@@ -0,0 +1,68 @@
+{
+  "type": "insert_abbreviation",
+  "test_cases": [
+	{
+      "class": "AbbreviationInsertion",
+      "inputs": {
+        "sentence": "Make sure you've gone online to download one of the vouchers - it's definitely not worth paying full price for!"
+      },
+      "outputs": [{
+        "sentence": "Make sure U've gone ONL 2 D/L one of DA vouchers - it's DFNTLY not worth paying full price 4!"
```

Comment:
Also, won't it be better to use small case or at least have an argument in the constructor (i.e. configurable case)?

Reply:
I mean "DA" can easily be a full form.. but not "da"..isn't it?